File: blog-post-2/cpp/ReverseNumber.cpp
#include <iostream>
using namespace std;

int main()
{
    int num,reverse,rm;
    cout << "Enter number: ";
    cin >> num;
    reverse = 0;
    while (num > 0)
    {
        rm = num % 10;
        reverse = (reverse * 10) + rm;
        num /= 10;
    }
    cout << "reverse number: " << reverse;
    return 0;
}

File: blog-post-2/kotlin/ReverseNumber.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var scanner = Scanner(System.`in`)
    print("Enter number: ")
    var num = scanner.nextInt()
    var reverse = 0
    while (num > 0){
        val rm = num % 10
        reverse = (reverse * 10) + rm
        num /= 10
    }
    println("reverse number: "+reverse)
}